### DISCUSSION TITLE
Disable java linting

### DISCUSSION THREAD

**Discussion by Petrprogs** [redhat-developer/vscode-java#3217] [Q&A]
How to disable lint and use only autocorrect and autocomplete?

##### Environment
- Operating System: Debian 12 x64
- JDK version: OpenJDK  20
- Visual Studio Code version: 1.78.0
- Java extension version: 1.20.0

##### Steps To Reproduce

1.  Open any java project
2.  The entire project is linted which is not necessary for me.

**Comment by fbricon** [comment]
Set `java.server.launchMode` to `LightWeight`. As per the doc in the [readme](https://github.com/redhat-developer/vscode-java#supported-vs-code-settings):

> Starts a syntax server with lower start-up cost. Only provides syntax features such as outline, navigation, javadoc, syntax errors. The lightweight mode won't load thirdparty extensions, such as java test runner, java debugger, etc.
